src/main/reactjs/src/styles/components/dialog/_dialog.scss

```diff
@@ -0,0 +1,11 @@
+.dialog__success {
+  .MuiDialog-paper {
+    border-top: 1vh $color-green-600 solid;
+  }
+}
+
+.dialog__error {
+  .MuiDialog-paper {
+    border-top: 1vh $color-red-500 solid;
+  }
+}
```

Comment:
Kannattaisi laittaa nämä kaikki .dialog in alle. CSS:ssä material UI:n classien käyttö aiheuttaa ongelmia. Kannattaisi välttää niitä ja sen sijaan luoda omia classeja

Reply:
Siirsin .dialog alle ja sinne tuli vielä kolmantena luokkana `dialog__neutral`. En saanut tyylejä tarttumaan käyttämättä tuota .MuiDialog-paper -luokkaa, joten jätin vielä tuollaisiksi.

Reply:
Käyttäisin esim:

`@mixin top-border-color($color) {
  border-top: 1rem $color solid;
}

.dialog {
  &__success[role='dialog'] {
    @include top-border-color($color-some-some);
  }

  &__error[role='dialog'] {
    @include top-border-color($color-some-some);
  }

  &__info[role='dialog'] {
    @include top-border-color($color-some-some);
  }
}
`

dialog__info on parempi ehkä kuin dialog__neutral?

Reply:
Nice, toi role-attribuutti vaikuttaa hyvältä. `dialog__info` myös parempi kuin `dialog__neutral`.

Reply:
Ei ihan vielä tuollaisenaan toiminut, piti vielä muuttaa `[role='dialog']` noiden luokkien descendantiksi. Poistin myös border-radiukset, kun wireframeissakaan niitä ei ollut.